src/Algolia.Search/Transport/HttpTransport.cs

```diff
@@ -48,6 +48,7 @@ internal class HttpTransport
         private readonly ISerializer _serializer;
         private readonly RetryStrategy _retryStrategy;
         private readonly AlgoliaConfig _algoliaConfig;
+        private string error;
```

Comment:
Yep that's was my second choice. But because I wasn't sure if we retrieve the `error.Message` or the `error` . I choose this name. I'll update it :)